packages/svelte/tests/runtime-legacy/shared.ts

```diff
@@ -303,7 +319,7 @@ async function run_test_variant(
 							htmlEqualWithOptions: assert_html_equal_with_options
 						},
 						variant,
-						component: instance,
+						component: runes ? props : instance,
```

Comment:
this will surprise us down the line when we want to invoke actual exports on the instance. We should merge the objects instead

Reply:
Adjusted that

Reply:
The adjustments also ensure that I don't have to wrap every usage site with `flushSync` which I think is better.